app/styles/_animations.scss

```diff
@@ -0,0 +1,24 @@
+@keyframes dotGrow {
+  20% {
+    transform: scale(1);
+  }
+
+  50% {
+    transform: scale(1.5);
+    opacity: 1;
+  }
+
+  80% {
+    transform: scale(1);
+  }
+}
+
+@keyframes rotating {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  100% {
+    transform: rotate(360deg);
+  }
+}
```

Comment:
These are fun! Where is `dotGrow` used? I see you moved it from `_loaders.scss` here, but it must only be used elsewhere in the site since I don't see it referenced here.

If it's not used, all is good, actually, but if it is we should be sure that whatever makes use of it is including `_animations.scss` now, rather than trying to find it from `_loaders.scss`.

Reply:
Yes, I'm including the all animations here so they can be reused in different components. This one's still being used in `_loaders.scss`

Reply:
Oh, I should have looked at the `.scss` include structure. Now I see this is more of a "lateral" move and **both** `_animations.scss` and `_loaders.scss` are included in `main.scss`. 👍